packages/scss/src/components/_container.scss

```diff
@@ -10,7 +10,7 @@
 		}
 	}
 
-	@media (max-width: _component("container.min-width", true)) {
+	@media (max-width: _component("container.min-width", true) - 1) {
```

Comment:
Written this way, isn't it presupposing that value are in px, not in any other unit. It might be dangerous

Reply:
I'd just like to propose that we turn this into `em`. ;)
I'll add a check just in case!